subsys/net/lib/nrf_cloud/src/nrf_cloud_log.c

```diff
@@ -257,14 +265,15 @@ int nrf_cloud_log_send(int log_level, const char *fmt, ...)
 		LOG_INF("Failed to take semaphore");
 		return err; /* Caller should try again; busy or other error */
 	}
-	err = nrf_cloud_coap_json_message_send(output.data.ptr);
+	err = nrf_cloud_coap_json_message_send(output.data.ptr, false);
```

Comment:
I think you've adjusted this signature in the wrong commit, possibly

Reply:
No, I meant to do it here, but if you are concerned that bisecting between these two commits will result in a build error, I can split into the two commits.

Reply:
Yeah, bisecting is my main concern

When you say split into two commits, I'm not 100% sure what you mean, are you talking about adding a 4th commit, or just moving this signature adjustment to the commit that adds bulk messages?

Reply:
move the signature adjustment to the other commit, but leave the remainder of changes to this file in this commit

Reply:
Gotcha, yeah, that would be a good idea IMO